tests/forms-schema/forms-schema-form-events.test.ts

```diff
@@ -111,6 +111,113 @@ describe('Payment events should throw errors when not passed to "paymentEvents"
   })
 })
 
+describe('CP Pay', () => {
```

Comment:
is this meant to be in this PR? No problems with it, just checking as its unrelated

Reply:
yes, these tests are copied from the API. I will be removing them from the API when I updated the SDK.